### PR TITLE
Replace LabelBinarizer by label_binarize

### LINKED ISSUE #108: [DOCUMENTATION] Wrong import on github readme example
**Is your documentation request related to a problem? Please describe.**

On the main site, on the example you have 

from maple.estimators import MapieRegressor

but I had an error message saying that estimators was not found. Then I went inside your codes and you have

from mapie.regression import MapieRegressor

This is working. 

**Describe the solution you'd like**
You may want to update the main site.

### LINKED ISSUE #116: [BUG] When `cv = "prefit"`, number of classes between calibration set and fitted model can be different
**Describe the bug**
For `MapieClassifier`, when `cv` is set to "prefit", the calibration set does not necessarily contain all the classes of the original dataset. Therefore, the base classifier could have been trained on a training set containing a larger number of classes. This discrepancy triggers a bug for the `cumulated_score` in the `fit` method.

**To Reproduce**
Simply run a simulation with a toy dataset in which the calibration set does not contain all the classes of the original dataset, with `cv="prefit"` and `method="cumulated_score"`. 

**Expected behavior**
This discrepancy can be fixed by using `sklearn.preprocessing.label_binarize` instead of ` sklearn.preprocessing.LabelBinarizer` in order to explicitely list the number of classes when "binarizing" the labels.

### PR DESCRIPTION
**Pull request by vtaquet**
# Description

Replace `LabelBinarizer` by `label_binarize` in the "cumulated_score" method to explicitely include the classes predicted by the base classifier. 

For MapieClassifier, when cv is set to "prefit", the calibration set does not necessarily contain all the classes of the original dataset. Therefore, the base classifier could have been trained on a training set containing a larger number of classes. This discrepancy triggers a bug for the cumulated_score in the fit method.

Fixes #116 
Fixes #108 

## Type of change

Please remove options that are irrelevant.

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

All existing unit tests + New test checking that the base prefitted classifier has the `classes_` attribute.

# Checklist:

- [x] I have read the [contributing guidelines](https://github.com/simai-ml/MAPIE/blob/master/CONTRIBUTING.rst)
- [x] I have updated the [HISTORY.rst](https://github.com/simai-ml/MAPIE/blob/master/HISTORY.rst) and [AUTHORS.rst](https://github.com/simai-ml/MAPIE/blob/master/AUTHORS.rst) files
- [x] Linting passes successfully : `make lint`
- [x] Typing passes successfully : `make type-check`
- [x] Unit tests pass successfully : `make tests`
- [x] Coverage is 100% : `make coverage`
- [x] Documentation builds successfully : `make doc`

### CODE REVIEW

File: mapie/tests/test_classification.py [hunk 1]
```diff
@@ -819,6 +822,7 @@ def test_sum_proba_to_one_fit(y_pred_proba: ArrayLike) -> None:
     Test if when the output probabilities of the model do not
     sum to one, return an error in the fit method.
     """
+    print(y_pred_proba)
```
Comment (vincentblot28) [inline mapie/tests/test_classification.py:825]: Remove print

**Review by gmartinonQM — APPROVED** [review]
LGTM !

File: mapie/tests/test_classification.py (hunk 1 above)
Comment (vtaquet) [reply mapie/tests/test_classification.py]: Removed !
